tests/jobs/test_backfill_job.py

```diff
@@ -147,31 +147,25 @@ def test_dag_run_with_finished_tasks_set_to_success(self, dag_maker):
 
         assert State.SUCCESS == dag_run.state
 
-    @pytest.mark.xfail(condition=True, reason="This test is flaky")
     @pytest.mark.backend("postgres", "mysql")
     def test_trigger_controller_dag(self):
```

Comment:
```suggestion
    def test_trigger_controller_dag(self, session):
```

you can do this to get rid of the `session = settings.Session()` line.